gradle/jacoco.gradle

```diff
@@ -7,4 +7,5 @@ jacoco {
 tasks.withType(Test) {
   // needed to capture coverage from Robolectric tests
   jacoco.includeNoLocationClasses = true
+  jacoco.excludes = ['jdk.internal.*']
```

Comment:
Why is this needed here? I imagine you were running into some problems with unit tests and Java 11?

Reply:
Yeah let me try a different Java version and see if that fixes the problem

Reply:
Vitaly mentioned that he'll help to drive https://github.com/mapbox/mapbox-navigation-android/pull/4454 to the finish line so let's remove this change from here for now.